code/game/machinery/_machines_base/stock_parts/shielding.dm

```diff
@@ -9,21 +9,21 @@
 	name = "fuse box"
 	icon_state = "fusebox"
 	desc = "A bloc of multi-use fuses, protecting the machine against the electrical current spikes."
-	protection_types = list(ELECTROCUTE)
+	protection_types = list(DAMAGE_SHOCK)
 	material = /decl/material/solid/metal/steel
 	matter = list(/decl/material/solid/fiberglass = MATTER_AMOUNT_REINFORCEMENT)
 
 /obj/item/stock_parts/shielding/kinetic
 	name = "internal armor"
 	icon_state = "armor"
 	desc = "Kinetic resistant armor plates to line the machine with."
-	protection_types = list(BRUTE)
+	protection_types = list(DAMAGE_BRUTE)
 	material = /decl/material/solid/metal/steel
 
 /obj/item/stock_parts/shielding/heat
 	name = "heatsink"
 	icon_state = "heatsink"
 	desc = "Active cooling system protecting machinery against the high temperatures."
-	protection_types = list(BURN)
+	protection_types = list(DAMAGE_BURN)
```

Comment:
Looking at the code that uses this far, this doesn't actually need to be a list.
More of a nitpick than an actual change request.